README.md

```diff
@@ -13,6 +13,7 @@
     - [Prerequisites](#prerequisites)
     - [Setup](#setup)
     - [Install](#install)
+    - [Usage](#usage)
   - [👥 Authors ](#-authors-)
```

Comment:
- [ ] Kindly fix the linkedln broken link in the Author section of the README file; this wil make the README file to be more engaging and informative. 👍